.github/workflows/test.yml

```diff
@@ -91,8 +91,6 @@ jobs:
           go-version: '1.15'
 
       - name: promtool test
-        env:
```

Comment:
I think we should restore this to validate cartridge metrics names (like cartridge issues)